### PR TITLE
Update contractkit to 1.0.2

### LINKED ISSUE #65: User not able to receive Attestation codes when user try to complete signup flow through Restore my account section.
**Frequency:** 100%
**Repro on build version:** Android Internal Build V1.11.0(1004294337), Test Flight Build V1.11.0(47)
**Does not Repro on build version:** Playstore build V1.10.1, App store build V1.10.1,
**Repro on devices:** iPhone 6 + (12.4.5), Samsung galaxy J7 pro (8), iPhone 6s (14.2), iPhone XR (14.4)
Testing Account: [Backup Key](https://docs.google.com/spreadsheets/d/1Iy41GGA6e2pES2XHjBn7nb8X3n8zZsu7EW1jvwD2e70/edit#gid=0&range=B5) 
  **Repro Steps:**

1. Launch the application 
2. Go through tutorials 
3. Tap on Restore my account button
4. enter name & tap on next button 
5. Enter valid account account key to restore 
6. Tap on restore button
7. Enter any valid phone number 
8. Complete captcha 
9. Wait for 2 mins time 
10. Observe 

**Current Behavior:** User not received any Attestation code 
**Expected Behavior:** user should receive the codes without any verification fail. 
**Observation:** 

1. Verification failed error message is shown when user try to complete signup flow.
2. From Confirm CTA: User not able to receive codes when go through CTA.

**Impact:** User get blocked to complete the signup flow .
**Attachment:**  [NotReceivedAttestation.mp4](https://drive.google.com/file/d/1-MSAtoDK2yDQ5jcGt_nhYHgAuExha-wF/view?usp=sharing)

### PR DESCRIPTION
**Pull request by i1skn**
### Description
Updates contractkit to 1.0.2. 

This is important because after the split of the monorepo we've used set contraktkit to`1.0.1` instead of `1.0.1-dev` and the latter had important changes in it.

### Tested

Yes

### Related issues

- Fixes #65 

### Backwards compatibility

Yes

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1294,6 +1299,18 @@
...
+  integrity sha512-kCeOmSU+f+ufb8G2VU2UnOS28QznooBUt5Vgzvg7+ZoUmTB7PBxqPSw4ViQdRxSAMi1yZihTx/HX9PijZ1Mptw==
+  dependencies:
+    "@celo/utils" "1.0.2"
+    "@types/debug" "^4.1.5"
+    "@types/utf8" "^2.1.6"
+    bignumber.js "^9.0.0"
+    debug "^4.1.1"
+    utf8 "3.0.0"
+
 "@celo/contractkit@1.0.1":
```
Comment (jeanregisser) [inline yarn.lock:1314]: Looks like another package still references `contractkit@1.0.1`, would be nice to upgrade them all.

File: yarn.lock [hunk 2]
```diff
@@ -1384,6 +1419,40 @@
...
+    fp-ts "2.1.1"
+    google-libphonenumber "^3.2.15"
+    io-ts "2.0.1"
+    keccak256 "^1.0.0"
+    lodash "^4.17.14"
+    numeral "^2.0.6"
+    web3-eth-abi "1.3.0"
+    web3-utils "1.3.0"
+
 "@celo/wallet-base@1.0.1":
```
Comment (jeanregisser) [inline yarn.lock:1456]: ditto about other packages referencing older versions.

File: yarn.lock [hunk 3]
```diff
@@ -1399,6 +1468,21 @@
...
+    "@celo/connect" "1.0.2"
+    "@celo/utils" "1.0.2"
+    "@types/debug" "^4.1.5"
+    "@types/ethereumjs-util" "^5.2.0"
+    bignumber.js "^9.0.0"
+    debug "^4.1.1"
+    eth-lib "^0.2.8"
+    ethereumjs-util "^5.2.0"
+
 "@celo/wallet-local@1.0.1":
```
Comment (jeanregisser) [inline yarn.lock:1486]: ditto

**Review by jeanregisser — COMMENTED** [review]
Thanks!

**Comment by i1skn** [comment]
@jeanregisser I see from `yarn why` that packages like `@celo/blockchain-api` and `@celo/notification-service` are still dependent on older version, but I'm a bit hesitant to update them as a part of this PR. What do you think if we create an issue to update the rest of internal dependencies to the latest versions?

**Comment by jeanregisser** [comment]
Yes fine to update in a separate PR.

**Review by jeanregisser — APPROVED** [review]

**Comment by Celoqa** [comment]
Hi @i1skn We have verified this issue using latest Android Internal Build V1.12.0(1004294340) & Test flight build V1.12.0(49) and found below issues is fixed.
- User is able to receive Attestation codes when user try to complete signup flow through Restore my account section. 
**Fixed issue:**
- #65

**Pull request merged**